app/src/main/java/com/criptext/monkeychatandroid/MainActivity.java

```diff
@@ -352,9 +352,27 @@ public void setStatusBarState(Utils.ConnectionStatus status){
 
     /**
      * Update a status message. This is normally used after you send a message.
+     * @param oldId message old id
      * @param id message id
      * @param newStatus new status to change
      */
+    private void updateMessage(final String oldId, final String id, final MonkeyItem.DeliveryStatus newStatus) {
```

Comment:
Please update a new version of this method in which doesn't use MonkeyItemTransaction.